src/number_theory/number_field/embeddings.lean

```diff
@@ -230,7 +231,7 @@ instance : nonneg_hom_class (infinite_place K) K ℝ :=
   coe_injective' := λ _ _ h, subtype.eq (absolute_value.ext (λ x, congr_fun h x)),
   map_nonneg := λ w x, w.1.nonneg _ }
 
-lemma coe_mk (φ : K →+* ℂ) : mk φ = place φ := rfl
+lemma coe_mk (φ : K →+* ℂ) : (mk φ).1 = place φ := rfl
```

Comment:
This should use the arrow I commented with earlier, not `.1`; the latter is not the simp-normal form.

Reply:
Oh nevermind, that doesn't work...

Reply:
```suggestion
@[simp] lemma val_mk (φ : K →+* ℂ) : (mk φ).val = place φ := rfl
lemma coe_mk (φ : K →+* ℂ) : ⇑(mk φ) = place φ := rfl
```